tombstone_gc.cc

```diff
@@ -118,25 +100,67 @@ tombstone_gc_state::get_gc_before_for_range_result tombstone_gc_state::get_gc_be
                 }
             }
             if (hits == 0) {
-                min_repair_timestamp = gc_clock::time_point::min();
-                max_repair_timestamp = gc_clock::time_point::min();
+                min_reconcile_timestamp = gc_clock::time_point::min();
+                max_reconcile_timestamp = gc_clock::time_point::min();
             } else {
                 knows_entire_range = hits == 1 && contains_all;
-                min_repair_timestamp = min;
-                max_repair_timestamp = max;
+                min_reconcile_timestamp = min;
+                max_reconcile_timestamp = max;
             }
-            min_gc_before = check_min(s, saturating_subtract(min_repair_timestamp, propagation_delay));
-            max_gc_before = check_min(s, saturating_subtract(max_repair_timestamp, propagation_delay));
+            min_gc_before = check_min(s, saturating_subtract(min_reconcile_timestamp, propagation_delay));
+            max_gc_before = check_min(s, saturating_subtract(max_reconcile_timestamp, propagation_delay));
         };
-        dblog.trace("Get gc_before for ks={}, table={}, range={}, mode=repair, min_repair_timestamp={}, max_repair_timestamp={}, propagation_delay={}, min_gc_before={}, max_gc_before={}, hits={}, knows_entire_range={}",
-                s->ks_name(), s->cf_name(), range, min_repair_timestamp, max_repair_timestamp, propagation_delay.count(), min_gc_before, max_gc_before, hits, knows_entire_range);
-        return {min_gc_before, max_gc_before, knows_entire_range};
+        dblog.trace(
+            "Get gc_before for ks={}, table={}, range={}, mode=reconcile, min_reconcile_timestamp={}, max_reconcile_timestamp={}, propagation_delay={}, "
+            "min_gc_before={}, max_gc_before={}, hits={}, knows_entire_range={}",
+            s->ks_name(),
+            s->cf_name(),
+            range,
+            min_reconcile_timestamp,
+            max_reconcile_timestamp,
+            propagation_delay.count(),
+            min_gc_before,
+            max_gc_before,
+            hits,
+            knows_entire_range);
```

Comment:
yeah, it's again the automatic formatting according to the current .clang-format ... seems when the line would cross the max line length threshold it will reformat to put each parameter on a separate line

Reply:
I opened https://github.com/scylladb/scylladb/issues/20951